File: extensions/youtube/src/main/java/app/revanced/extension/youtube/patches/ZoomHapticsPatch.java
package app.revanced.extension.youtube.patches;

import app.revanced.extension.youtube.settings.Settings;

@SuppressWarnings("unused")
public class ZoomHapticsPatch {
    public static boolean shouldVibrate() {
        return !Settings.DISABLE_ZOOM_HAPTICS.get();
    }
}

File: patches/src/main/kotlin/app/revanced/patches/youtube/player/zoomhaptics/ZoomHapticsPatch.kt
package app.revanced.patches.youtube.player.zoomhaptics

import app.revanced.patcher.patch.bytecodePatch
import app.revanced.patches.youtube.player.hapticfeedback.disableHapticFeedbackPatch

@Deprecated("Superseded by disableHapticFeedbackPatch", ReplaceWith("disableHapticFeedbackPatch"))
val zoomHapticsPatch = bytecodePatch(
    description = "Adds an option to disable haptics when zooming.",
) {
    dependsOn(disableHapticFeedbackPatch)
}
